docs/devguide/newbeat.asciidoc

```diff
@@ -123,11 +123,12 @@ mkdir ${GOPATH}/src/github.com/{user}
 cd ${GOPATH}/src/github.com/{user}
 --------------------
 
-Run the mage script to generate the custom beat:
+If you haven't installed Mage on your system already, be sure to follow the [installation instructions here](https://magefile.org/). Next up we'll run a mage script to generate the custom beat:
 
 
 [source,shell]
 --------------------
+cd ${GOPATH}/src/github.com/elastic/beats
```

Comment:
Thank you! Sorry this got a bit delayed 😬. I'm going to remove the `cd` step as it's already included in the previous shell example on L123, fix the link from MD to Asciidoc, and change the wording around a little bit 😁 

```suggestion
Run the Mage script to generate the custom beat.
If you haven't installed Mage yet, see the https://magefile.org/[installation instructions].

[source,shell]
--------------------
```